build-tools/scripts/msbuild.mk

```diff
@@ -37,4 +37,11 @@ _BROKEN_MSBUILD := $(shell if ! pkg-config --atleast-version=4.8 mono ; then ech
 ifeq ($(_BROKEN_MSBUILD),Broken)
 MSBUILD_FLAGS += /p:_XAFixMSBuildFrameworkPathOverride=True
 endif   # $(_BROKEN_MSBUILD) == Broken
+else    # $(MSBUILD) != 1
+_CSC_EMITS_PDB  := $(shell if pkg-config --atleast-version=4.9 mono ; then echo Pdb; fi )
+ifeq ($(_CSC_EMITS_PDB),Pdb)
+MSBUILD_FLAGS += /p:_DebugFileExt=.pdb
```

Comment:
Are we setting this on the command line, in case we are building with xbuild? If yes, then we should do this only in that particular case.